Gemfile

```diff
@@ -61,6 +61,7 @@ gem 'will_paginate', '>= 3.0.6'
 gem 'will_paginate-bootstrap', '>= 1.0.1'
 gem 'georuby', '2.0'
 gem 'geokit-rails'
+gem 'json'
```

Comment:
Oops, is this required? Also, could you remove the Gemfile.lock from the PR as well? Thanks!